### PR TITLE
Compute the pod CIDR value for AWS provider

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

**Comment by whites11** [comment]
Can you provide some info about why this change is needed? I don't get it

**Comment by pipo02mix** [comment]
sure, a customer found it was not render anymore after our refactorization. The flag was there but not injected in the object. So you could pass `--pods-cidr` as an argument but it was not take into account on the output. Now it is

**Review by whites11 — APPROVED** [review]
LGTM

**Review by axbarsan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Changed
 
 - Make the `login` command validate the current context before considering it good to go.
+- Compute pod CIDR flag for AWS provider.
```
Comment (marians) [inline CHANGELOG.md:13]: To make this more descriptive for actual users:

```suggestion
- Re-enabled the `--pods-cidr` flag in the `template cluster` command.
```

**Review by marians — APPROVED** [review]

**Pull request merged**